### PR TITLE
feat: add ticket content

### PR DESCRIPTION
**Pull request by adifdwimaulana**

### CODE REVIEW

File: components/ticket/Content.vue [hunk 1]
```diff
@@ -1,18 +1,14 @@
...
-      <div class="hidden lg:block bg-gray-50 absolute top-0 bottom-0 left-3/4 w-screen" />
       <div class="mx-auto text-base max-w-prose lg:grid lg:grid-cols-2 lg:gap-8 lg:max-w-none">
         <div>
-          <h2 class="text-base text-indigo-600 font-semibold tracking-wide uppercase">
-            Case Study
-          </h2>
-          <h3 class="mt-2 text-3xl leading-8 font-extrabold tracking-tight text-gray-900 sm:text-4xl">
-            Meet Whitney
+          <h3 class="mt-2 text-3xl text-justify font-bold text-gray-500 sm:text-2xl">
+            PyCon Indonesia 2020 tickets are sold in the following types. Tickets are required to participate in the online conference.
```
Comment (ridhof) [inline components/ticket/Content.vue:7]: 2021 mas wkwkwk

File: components/ticket/Content.vue [hunk 2]
```diff
@@ -46,37 +42,57 @@
...
... the Cash on Delivery (COD) system. So you have to pay your shipping fee when your merchandise package has come to your address. Our merchandise will be produced at Surabaya City, East Java. You can choose to send your merchandise from <span class="font-bold text-gray-800">Surabaya</span> or one of these cities: <span class="font-bold text-gray-800">Bandung</span>, <span class="font-bold text-gray-800">Yogyakarta</span>, <span class="font-bold text-gray-800">Bogor</span>, and <span class="font-bold text-gray-800">Makassar</span>. Choosing origin cities other than Surabaya will cost more days on the delivery process. For shipping fee please check at <a href="https://www.jet.co.id/rates" target="_blank" class="font-bold text-gray-900">J&amp;T</a> Express, the merchandise package's weight is approximately 1 kilogram. For international participants or overseas shipping addresses, you can still get your merchandise but you have to pay for your shipping first. Please contact pycon@python.or.id to discuss your shipping fee. We will use <a href="https://www.posindonesia.co.id/en/" target="_blank" class="font-bold text-gray-900 no-underline">POS Indonesia</a> for international shipment.</p>
```
Comment (ridhof) [inline components/ticket/Content.vue:57]: sama nih

File: components/ticket/Content.vue [hunk 3]
```diff
@@ -46,37 +42,57 @@
...
-export default {}
+export default {
+  data () {
+    return {}
+  },
+  computed: {
+    tickets () {
+      return [
+        { type: 'Regular Ticket', description: 'This ticket includes event access for two days. We will send the link for the event to your registered email address one week before the event.', image: 'Regular.jpg' },
+        { type: 'Merchandise Ticket', description: 'This ticket includes event access for two days and PyCon ID 2020 Exclusive Merchandise. We will send the link for the event to your registered email address one week before the event. We will send your merchandise in about one week before the event and we will inform your AWB via your registered email.', image: 'Merchandise.jpg' },
```
Comment (ridhof) [inline components/ticket/Content.vue:92]: sama

File: components/ticket/Content.vue [hunk 4]
```diff
@@ -46,37 +42,57 @@
...
+        { type: 'Patron Ticket', description: 'Be a Patron by supporting this event. Your name will be listed on Patron Section. This ticket includes event access for two days and PyCon ID 2020 Exclusive Merchandise. We will send the link for the event to your registered email address one week before the event. We will send your merchandise in about one week before the event and we will inform your AWB via your registered email.', image: 'Patron.jpg' }
```
Comment (ridhof) [inline components/ticket/Content.vue:93]: +1

File: components/ticket/Header.vue [hunk 5]
```diff
@@ -0,0 +1,20 @@
+<template>
+  <div class="bg-info-800">
+    <div class="mx-auto py-16 px-4 sm:py-24 sm:px-6 lg:px-8 lg:flex lg:justify-between">
+      <div class="w-full">
+        <h2 class="text-4xl text-center font-extrabold text-white sm:text-5xl sm:tracking-tight lg:text-6xl">
+          Ticket
+        </h2>
+        <p class="mt-5 text-xl text-center text-gray-200">
+          PyCon Indonesia 2020 tickets are sold in the following types
```
Comment (ridhof) [inline components/ticket/Header.vue:9]: +1

**Review by ridhof — COMMENTED** [review]
emang sengaja 2020 kah mas @adifdwimaulana ?

File: components/ticket/Content.vue (hunk 1 above)
Comment (adifdwimaulana) [reply components/ticket/Content.vue]: fixed mas

**Review by ridhof — APPROVED** [review]
LGTM!

**Pull request merged**